luigi/target.py

```diff
@@ -128,11 +128,6 @@ def mkdir(self, path, parents=True, raise_if_exists=False):
                              exist, raise luigi.target.MissingParentDirectory
         :param bool raise_if_exists: raise luigi.target.FileAlreadyExists if
                                      the folder already exists.
-
-        *Note*: This method is optional, not all FileSystem subclasses implements it.
-
-        *Note*: parents and raise_if_exists were added in August 2014. Some
-                implementations might not support these flags yet.
```

Comment:
A note for however is curious.

I think these two notes add negative value only. In general we want people to implement this method (in particular after this patch). If we leave these old (and currently mis-rendered) notes, file system implementors might not implement this method referring to this.